content/learning-paths/smartphones-and-mobile/afrc/how-to-2.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How to run the code examples
+weight: 3
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## How to run the code examples
+
+This Learning Path assumes you are familiar with the Vulkan API and have an application that uses it to create and render images on Android. If that is the case, you may skip this step.
+
+Otherwise, you may use a test application like those provided by the [Khronos Vulkan Samples repository](https://github.com/KhronosGroup/Vulkan-Samples).
+
+{{% notice Note %}}
+You do not need to create a new Vulkan Sample to complete this Learning Path, you may famliarise yourself with the code examples and save them as a reference.
+There is already a Vulkan Sample that allows you to experiment with AFRC without any coding required, see the [Image Compression Control](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/samples/performance/image_compression_control/README.adoc) performance sample.
+{{% /notice %}}
+
+### Build Khronos' Vulkan Samples
+
+[Download](https://github.com/KhronosGroup/Vulkan-Samples?tab=readme-ov-file#setup) the repository:
+
+```bash
+git clone --recurse-submodules https://github.com/KhronosGroup/Vulkan-Samples.git
+cd vulkan-samples
+python ./scripts/generate.py android
+```
+
+Follow the [build instructions](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/docs/build.adoc#android) to run the example applications on an Android device.
+
+{{% notice  %}}
+You may use [Android Studio](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/docs/build.adoc#build-with-android-studio) to more easily build and run the samples, the build instructions explain how.
+{{% /notice %}}
+
+### Create a test sample
+
+We can use the default [API sample template](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/scripts/README.adoc#generate-api-sample) to write and run the code snippets presented in this Learning Path.
```

Comment:
We have the note above, but should we also have a first sentence here? "If you are coding an example rather than following along with the existing Image Compression sample, we need to first create a test sample. We can use..." or something like that?